joomla/joomla-cms
# Utilize hint attribute in JFormFieldList

### Summary of Changes

Currently we ignore the hint attribute for List type form field. This PR is to utilize it if provided to add placeholder (as well as data-palceholder for chosen and select2) attributes to the rendered input control.
### Testing Instructions

Create a form field for type list using xml in any component, provide the hint attribute as we do for text type fields. The hint should be visible when no options are selected on the UI.

``` xml
        <field
            name="test_list"
            type="list"
            multiple="true"
            hint="Test Hint for multiple"
        />
        <field
            name="test_list_single"
            type="list"
            hint="Test Hint for single"
        />
```

Without this PR, the hint was ignored.

Also test with chosen, select2.
### Documentation Changes Required

We need to add the information about this attribute (hint) being available in `list` field also.
